src/server_socket.cpp

```diff
@@ -320,7 +328,6 @@ void AOApplication::_p_handle_server_packet(DRPacket p_packet)
   }
   else if (l_header == "CL")
   {
-    qDebug() << l_content;
```

Comment:
Since it's visible here, let's add a boundary check for the content of the clock to avoid a possible out of bounds scenario.